### PR TITLE
[Bugfix] Fix the title view appearance for unnamed users

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
## Purpose

* Changed the background color for the title view when a user's name is empty and a microphone isn't muted.
Incorrect_background
No_background_shown

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
1. Connect to a meeting 
2. Connect to the same meeting on an another device without specifying your "Displya Name"
3. Unmute your microphone on the device from step 2
4. Check the title view on the device from the step 1
```

## What to Check
Verify that the following are valid
* a user's name and a mute status are shown correctly

### CODE REVIEW

File: AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/Grid/Cell/ParticipantGridCellVideoView.swift [hunk 1]
```diff
@@ -28,7 +31,7 @@ struct ParticipantGridCellVideoView: View {
                                  mutedIconSize: 14)
                 .padding(.vertical, 2)
                 .padding(.horizontal, 4)
-                .background(Color(StyleProvider.color.overlay))
+                .background(isTitleViewEmpty ? .clear : Color(StyleProvider.color.overlay))
```
Comment (AmyL219) [inline AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/Grid/Cell/ParticipantGridCellVideoView.swift:34]: Feel like it make more sense to move  `isTitleViewEmpty` to `ParticipantTitleView` and check if participantTitleView should remove the background dot

Comment (vhuseinova-msft) [reply AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/Grid/Cell/ParticipantGridCellVideoView.swift:34]: As disscused with @AmyL219:
ParticipantTitleView is used in a couple of other locations without any background color and paddings, and this check should be appplied only for this case.

Comment (vhuseinova-msft) [reply AzureCommunicationUI/AzureCommunicationUI/Presentation/SwiftUI/Calling/Grid/Cell/ParticipantGridCellVideoView.swift:34]: As disscused with @AmyL219, moved the padding setup to ParticipantTitleView

**Review by AmyL219 — APPROVED** [review]

**Review by kevinyulu — APPROVED** [review]

**Comment by AmyL219** [comment]
> @AmyL219 for unnamed participant, their name appears as "Unnamed Participant" in the participant list drawer. Just checking if we need check UX and follow the same naming for unnamed participant in this case. From this PR it looks like the participants would have no name labels if they turned the video on and stay unmuted. cc @vhuseinova-msft

ParticipantList should be bug free right now. For name labels, I think make it empty is expected. Similar to Teams behavior.

**Pull request merged**